WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -3,10 +3,11 @@ import Gridicons
 import SVProgressHUD
 import WordPressShared
 import WPMediaPicker
+import MobileCoreServices
 
 /// Displays the user's media library in a grid
 ///
-class MediaLibraryViewController: UIViewController {
+class MediaLibraryViewController: UIViewController, UIDocumentPickerDelegate {
```

Comment:
We typically like to split out protocol conformances into their own extensions, just to keep things split up logically. So, this could move to an extension further down the file:

```
// MARK: - UIDocumentPickerDelegate

extension MediaLibraryViewController: UIDocumentPickerDelegate {
  methods go here...
}
```